modules/conductor/src/main/java/org/opencastproject/event/handler/AssetManagerUpdatedEventHandler.java

```diff
@@ -217,13 +217,21 @@ public void handleEvent(final SeriesItem seriesItem) {
             // setting the URI to a new source so the checksum will most like be invalid
             episodeCatalog.setChecksum(null);
           }
+          authorizationService.removeAcl(mp, AclScope.Series);
+          // here we don't know the series extended metadata types,
+          // we assume that all series catalog flavors have a fixed subtype: series
+          MediaPackageElementFlavor seriesFlavor = MediaPackageElementFlavor.flavor("*", "series");
```

Comment:
Identifying the catalogs that way is essentially a bug. As this idea is not introduced by this PR, I don't expect you to fix this, though. 

I've created https://opencast.jira.com/browse/MH-12769